src/api/libcellml/componententity.h

```diff
@@ -291,6 +291,13 @@ class LIBCELLML_EXPORT ComponentEntity: public NamedEntity
      */
     std::string encapsulationId() const;
 
+    /**
+     * @brief Remove the encapsulation Id for this entity.
+     *
+     * Clear the encapsulation identifier for this entity.
```

Comment:
Replace "identifier" with "Id" to be consistent with the rest of the documentation for this class (see `setEncapsulationId()` for instance).

Reply:
92dfea1